ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -82,6 +82,10 @@ def adaptive_avg_pool1d(x, output_size, name=None):
 def adaptive_avg_pool2d(x, output_size, data_format="NCHW", name=None):
     return ivy.adaptive_avg_pool2d(x, output_size)
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+def adaptive_avg_pool3d(x, output_size, data_format="NCHW", name=None):
+    return ivy.adaptive_avg_pool3d(x, output_size)
```

Comment:
Why did you implement `adaptive_avg_pool3d`? The issue that you linked to only attempts to implement `ihfft` so it would be better if you could create another issue and PR for this :+1:

Reply:
Hey @xoiga123 ,  adaptive_avg_pool3d is another issue, i will revert it, probably pushed it in my master branch.